### PR TITLE
[TRANSLATIONS] Turkish translation updates

### PR DESCRIPTION
**Pull request by eersoy93**
## Purpose

Update Turkish translations for ReactOS.

JIRA issue: None

## Proposed changes

- [DESK.CPL] Update Turkish translations
- [USERENV] Update Turkish translations
- [SYSSETUP] Update Turkish translations
- [HIVEDEF.INF] Update Turkish translations
- [SHELL32] Update Turkish translations (parital)

## TODO

- None. I am too busy for updating translations now.

### CODE REVIEW

File: dll/win32/userenv/lang/tr-TR.rc [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+/* TRANSLATOR: 2014, 2020 Erdem Ersoy (eersoy93) (erdemersoy [at] erdemersoy [dot] net) */
 
 LANGUAGE LANG_TURKISH, SUBLANG_DEFAULT
 
 /* See also: dll/win32/shell32/lang */
 STRINGTABLE
 BEGIN
     IDS_PROFILEPATH "%SystemDrive%\\Belgeler ve Ayarlar"
-    IDS_APPDATA "Uygulama Verisi"
+    IDS_APPDATA "Uygulama Verileri"
```
Comment (katahiromz) [inline dll/win32/userenv/lang/tr-TR.rc:9]: These changes are synchronized with `dll/win32/shell32/lang`?

Comment (eersoy93) [reply dll/win32/userenv/lang/tr-TR.rc:9]: Yes

**Comment by eersoy93** [comment]
I'm going to continue to update translations tomorrow.

**Comment by binarymaster** [comment]
Ok I made your PR a draft then. You can click "Ready for review" once it's ready.

**Comment by eersoy93** [comment]
I couldn't continue updating translations currently. You can apply these changes to upstream.

**Comment by eersoy93** [comment]
I am too busy for updating translations now. So, you can merge the PR. Sorry for my inconvience.

File: dll/cpl/desk/lang/tr-TR.rc [hunk 2]
```diff
@@ -154,79 +154,79 @@ END
...
     LTEXT "Yazı Tipi Boyutu", -1, 14, 20, 222, 9
     COMBOBOX IDC_FONTSIZE_COMBO, 14, 30, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 46, 223, 12
     GROUPBOX "Uyumluluk", -1, 7, 67, 237, 97
-    LTEXT "Eğer renk ayarlarını değiştirdiğinizden sonra bilgisayarınızı yeniden başlatmazsanız birtakım izlenceler yanlış çalışır.\r\n\nRenk ayarlarını değiştirdikten sonra:", -1, 12, 78, 228, 34
-    AUTORADIOBUTTON "&yeni renk ayarları uygulanmadan önce bilgisayarı yeniden başlat.", IDC_RESTART_RB, 14, 116, 228, 10
-    AUTORADIOBUTTON "y&eniden başlatmadan yeni renk ayarlarını uygula.", IDC_WITHOUTREBOOT_RB, 14, 131, 228, 10
-    AUTORADIOBUTTON "ye&ni renk ayarlarıuygulanmadan önce bana sor.", IDC_ASKME_RB, 14, 146, 228, 10
+    LTEXT "Eğer renk ayarlarını değiştirdiğinizden sonra bilgisayarınızı yeniden başlatmazsanız bazı programlar yanlış çalışır.\r\n\nRenk ayarlarını değiştirdikten sonra:", -1, 12, 78, 228, 34
+    AUTORADIOBUTTON "y&eni renk ayarları uygulanmadan önce bilgisayarı yeniden başlat", IDC_RESTART_RB, 14, 116, 228, 10
```
Comment (HBelusca) [inline dll/cpl/desk/lang/tr-TR.rc:166]: I've checked on english & other languages and these are full sentences starting with a capital letter and ending with a full stop.

Comment (eersoy93) [reply dll/cpl/desk/lang/tr-TR.rc]: Ok, I've updated these strings.

**Pull request merged**